File: app/src/main/java/com/semanientreprise/soundrecorderbox/MyApplicationClass.java
package com.semanientreprise.soundrecorderbox;

import android.app.Application;

import io.objectbox.BoxStore;

public class MyApplicationClass extends Application {

    public BoxStore boxStore;

    @Override
    public void onCreate() {
        super.onCreate();
        boxStore = MyObjectBox.builder().androidContext(MyApplicationClass.this).build();
    }

    public BoxStore getBoxStore() {
        return boxStore;
    }
}

File: app/src/main/java/com/semanientreprise/soundrecorderbox/MyApplicationClass.kt
package com.semanientreprise.soundrecorderbox

import android.app.Application
import android.content.Context
import androidx.multidex.MultiDex
import com.semanientreprise.soundrecorderbox.utils.ObjectBox

class MyApplicationClass : Application() {
    override fun onCreate() {
        super.onCreate()
        ObjectBox.init(this)
    }

    override fun attachBaseContext(base: Context) {
        super.attachBaseContext(base)
        MultiDex.install(this)
    }
}